### DISCUSSION TITLE
Remove earrings from Fooocus Masterpiece?

### DISCUSSION THREAD

**Discussion by jaimeastorga2000** [lllyasviel/Fooocus#3245] [Ideas]
I've noticed that the Fooocus Masterpiece style includes "earrings" in the prompt. This seems a little random, since there are lots of characters who should not necessarily have earrings (males, tomboys, etc.); it's not something that you would probably want in every image like "detailed eyes" or "perfect composition".

**Comment by CantainPike** [comment]
That is true, you can edit style file and delete it, but it will come back after each update
